modules/deploy/pages/deployment-option/cloud/monitor-cloud.adoc

```diff
@@ -0,0 +1,72 @@
+= Monitor Redpanda Cloud
+:description: Learn how to integrate the Redpanda Cloud metrics endpoint to monitor the health of your Dedicated or BYOC clusters.
+:page-cloud: true
+:env-cloud: true
+
+You can configure monitoring on your Redpanda Dedicated or BYOC cluster to maintain system health and optimize performance. Redpanda exports metrics from the xref:reference:public-metrics-reference.adoc[`/public_metrics`] Prometheus endpoint through the Admin API port (default: 9644). 
+
+== Configure monitoring
+
+You can monitor Redpanda with https://prometheus.io/[Prometheus^] or with any other monitoring and alerting tool, such as Datadog, New Relic, Elastic Cloud, Google, or Azure.
+
+[NOTE]
+====
+To maximize monitoring performance by minimizing the cardinality of data, some metrics are exported when their underlying features are in use, and are not exported when not in use. For example, a metric for consumer groups, xref:reference:public-metrics-reference.adoc#redpanda_kafka_consumer_group_committed_offset[`redpanda_kafka_consumer_group_committed_offset`], is not exported when no groups are registered.
+====
+
+=== Configure Prometheus
+
+To monitor a Dedicated or BYOC cluster in Prometheus:
+
+. On the Redpanda Cloud *Overview* page for your cluster, under *How to connect*, click the *Prometheus* tab. 
+
+. Click the copy icon for *Prometheus YAML* to copy the contents to your clipboard. 
++
+image::shared:cloud_metrics.png[How to connect screenshot]
+This provides the targets (that is, the hostnames for the endpoints for the running brokers) as well as the username and password for Prometheus:
++
+```
+- job_name: redpandaCloud-sample
+  static_configs:
+    - targets:
+        - console-321f60a4.cn6b1srsj6j1mdqdhil0.fmc.ppd.cloud.redpanda.com
+  metrics_path: /api/cloud/prometheus/public_metrics
+  basic_auth:
+    username: prometheus
+    password: "%kvjQr^E!xQ+lXqJ~g?5HhZ3"
+  scheme: https
+```
+
+. Save this information to Prometheus, and observe in Prometheus that metrics from Redpanda endpoints are scraped.
+
+TIP: For a sample Docker Compose file that brings up Prometheus and Grafana, see https://github.com/redpanda-data/observability/tree/main/cloud[Redpanda monitoring examples repository^]
+
+=== Configure DataDog
+
+The DataDog agent must be run in a container, and the container must be run in your own container infrastructure. Redpanda does not support launching this container inside a Dedicated or BYOC Kubernetes cluster. See the https://github.com/DataDog/integrations-extras/tree/master/redpanda[Redpanda DataDog integration].
+
+To build the DataDog container with the Redpanda integration:
```

Comment:
These steps aren't for _building_ the container, but configuring it. 

Might need someone who's more familiar with DataDog and Redpanda to validate this section (@hcoyote ?), but I think this needs tweaking. See subsequent comment below...